src/main/java/com/sandvoxel/generitech/common/tileentities/machines/TileEntityPulverizer.java

```diff
@@ -234,120 +249,116 @@ public void update() {
             }
         }
 
+        //for taking power/fuel from the block
         if ((this.container.takePower(powerUsage, true) == powerUsage && machineTier != MachineTier.TIER_0) || fuelRemaining > 0) {
             if (machineActive && !pulverizerPaused) {
                 ticksRemaining--;
 
                 if (machineTier != MachineTier.TIER_0)
                     this.container.takePower(powerUsage, false);
-                if (fuelRemaining > 0)
-                    fuelRemaining--;
+
 
             }
+        }
+        if (fuelRemaining > 0)
+            fuelRemaining--;
 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS00122)